src/main/resources/jenkins/branch/NoTriggerOrganizationFolderProperty/help-branches.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    Allows you to control the SCM commit trigger coming from branch indexing.
+    Supply a regular expression of branch names, for example <code>(?!release.*).*</code> or <code>PR-\d+</code>.
```

Comment:
Yes this is different from the syntax of `NamedExceptionsBranchPropertyStrategy.Named.name`, but regular expressions are more flexible, especially in this case where you may need to create a pattern in either the positive or negative sense.